pkgs/development/python-modules/world-bank-data/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  hatchling,
+  cachetools,
+  pandas,
+  requests,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "world-bank-data";
+  version = "0.1.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "world_bank_data";
+    inherit version;
+    hash = "sha256-UidtJovurzrZKWeI7n1bV0vluc5pSg92zKFELvZE9fw=";
+  };
+
+  build-system = [
+    hatchling
+  ];
+
+  dependencies = [
+    cachetools
+    pandas
+    requests
+  ];
+
+  doCheck = false;
+
+  nativeCheckInputs = [ pytestCheckHook ];
```

Comment:
Why is pytestCheckHook included if `doCheck = false`? Does it run anyway?

Reply:
Oops, I think it is because I first enabled the tests, but then all of them needed to do api calls to the internet. So I had to disable them and forgot to remove the hook